planner/physical.go

```diff
@@ -1,98 +1,34 @@
 package planner
 
-import (
-	"github.com/influxdata/flux"
-	"github.com/influxdata/flux/semantic"
-	"github.com/influxdata/platform"
-)
+import "github.com/influxdata/flux/plan"
 
-// PhysicalPlan represents a query tree/subtree whose nodes
-// are the physical operations/algorithms to be performed.
-type PhysicalPlan interface {
-	physical()
-	Predecessors() []PhysicalPlan
-	Successors() []PhysicalPlan
-	Cost() Cost
+// PhysicalProcedureSpec is similar to its logical counterpart but must provide a method to determine cost
+type PhysicalProcedureSpec interface {
+	Kind() plan.ProcedureKind
+	Copy() plan.ProcedureSpec
+	Cost(inStats []Statistics) (cost Cost, outStats Statistics)
```

Comment:
Why does `Cost()` produce `Statistics`? Should `PhysicalProcedureSpec` also have a `Statistics()` method?